bids_prov/spm_parser.py

```diff
@@ -248,11 +267,11 @@ def get_records(task_groups: dict, verbose=False) -> dict:
 
     for common_prefix_act, end_line_list in task_groups.items():
 
-        activity_id = "niiri:" + common_prefix_act + get_id()
+        activity_id = "urn:" + get_id()
         activity = {"@id": activity_id,
                     "label": format_activity_name(common_prefix_act),
                     "used": list(),
-                    "wasAssociatedWith": "RRID:SCR_007037",
+                    "wasAssociatedWith": "urn:" + agent_id,
```

Comment:
@thomasbtnfr: Have you tried here to use directly a UUID (without the conversion to string)? I would like to check how the namespace is handled by the uuid library (rather than having to add manually the `urn` namespace before).

Reply:
(If we add the urn namespace then we will have to declare it somewhere, in the context and in the RDF python library)

Reply:
Related to this issue : https://github.com/Inria-Empenn/BEP028_BIDSprov/issues/13

Reply:
urn does not need to be defined in the context, it's handled appropriately as an uri by most libraries i believe. 

https://api.dandiarchive.org/api/dandisets/000350/versions/0.221219.1506/assets/6b78d427-92e0-4a8e-bc64-481cbddea1e2/ (look at the publisher section). also we don't define it anywhere in our context: https://github.com/dandi/schema/blob/master/releases/0.6.3/context.json

Reply:
Thanks @satra! We opened https://github.com/Inria-Empenn/BEP028_BIDSprov/issues/13 to work on this in a coming PR. I'll link your comment in there!